examples/basic.py

```diff
@@ -0,0 +1,42 @@
+from javascript import require, On
```

Comment:
I think let's put it in a python folder, otherwise it'll be among the 15 other examples and not very visible (and a bit confusing)

Reply:
done